plugins/test/validate.py

```diff
@@ -68,8 +68,8 @@
 RETURN = """
   _raw:
     description:
-      - If data is valid return C(true)
-      - If data is invalid return C(false)
+      - If data is valid return C(True)
+      - If data is invalid return C(False)
```

Comment:
```suggestion
      - If data is valid return C(true)
      - If data is invalid return C(false)
```
Booleans in documentation should be lower-case.